Script/AtomicEditor/editor/Editor.ts

```diff
@@ -26,6 +27,9 @@ class Editor extends Atomic.ScriptObject {
 
         this.editorLicense = new EditorLicense();
 
+        var prefs = new Preferences();
+        prefs.read();
```

Comment:
https://github.com/rsredsq/AtomicGameEngine/commit/57dcff0fc5b3f06d7e93e150f189ea34a85f0a5b
Did it!